brig/cmd/brig/commands/run.go

```diff
@@ -13,16 +13,17 @@ import (
 )
 
 var (
-	runFile       string
-	runConfigFile string
-	runEvent      string
-	runPayload    string
-	runCommitish  string
-	runRef        string
-	runLogLevel   string
-	runNoProgress bool
-	runNoColor    bool
-	runBackground bool
+	runFile        string
+	runConfigFile  string
+	runEvent       string
+	runPayload     string
+	runJsonPayload string
```

Comment:
The older of these two variables, `runPayload`, references a file. The newer one `runJsonPayload`, is meant to directly contain the payload... but, the payload is JSON either way, so the "json" in the new variable `runJsonPayload` doesn't accurately disambiguate how each of these two similarly named variables are used. I would suggest rethinking the naming here and maybe changing the older variable to `runPayloadFile` and the newer one to `runInlinePayload`... or something like that.